src/screens/home/components/epoch/index.test.tsx

```diff
@@ -11,17 +19,38 @@ const mockI18n = {
   lang: 'en',
 };
 jest.mock('next-translate/useTranslation', () => () => mockI18n);
+jest.mock('@components', () => ({
+  Box: (props) => <div id="box" {...props} />,
+}));
+
+const mockEpoch = jest.fn().mockResolvedValue({
+  data: {
+    average_time: '0.535677777',
+    slot: '420000',
```

Comment:
This does not make sense. If `average_time` and `slot` are strings then why did you default them as numbers in the previous file?

```
const averageSlotTime = R.pathOr(0, ['average_slot_time_per_hour', 0, 'average_time'], data);
const slot = R.pathOr(0, ['block', 0, 'slot'], data);
```

Reply:
Oh because I default them as numbers at first but then changed to strings. Will change this file as well.

Reply:
My mistake, they should still be numbers in the test file.